### PR TITLE
Feature/ehgmsdk20/item give

### PR DESCRIPTION
**Pull request by ehgmsdk20**
Changes:

- Added item give function for item airdrop event

### CODE REVIEW

File: src/store.ts [hunk 1]
```diff
@@ -107,6 +107,17 @@ export default class Store extends AinftBase {
     return this.sendRequest(HttpMethod.DELETE, trailingUrl, query);
   }
 
+  giveItemToUser(appId: string, userId: string, itemName: string, quantity: number, reason?: string) {
```
Comment (woomurf) [inline src/store.ts:110]: Could you add a comment about this function as JsDoc?

Comment (woomurf) [inline src/store.ts:110]: minor) Could you add new lines between functions params?

Comment (ehgmsdk20) [reply src/store.ts]: Added, thanks!

Comment (ehgmsdk20) [reply src/store.ts]: Fixed, thanks!

**Comment by ehgmsdk20** [comment]
PTAL~ @woomurf

**Review by woomurf — APPROVED** [review]
LGTM, Thanks!

**Comment by ehgmsdk20** [comment]
Thanks for the review, now merging~

**Pull request merged**